prost-build/Cargo.toml

```diff
@@ -28,3 +28,8 @@ which = { version = "4", default-features = false }
 
 [dev-dependencies]
 env_logger = { version = "0.8", default-features = false }
+
+[workspace]
+members = [
+    "tests/single_include"
```

Comment:
This seems to be failing the ci build btw

Reply:
Yes, I did the bad thing. Think I undid the bad thing.